<launch>

    <node pkg="bullproof_perception" name="fake_camera" type="fake_camera.py"/>
    <!-- <node pkg="bullproof_perception" name="vision" type="vision.py"/> -->

</launch>